src/list.js

```diff
@@ -0,0 +1,46 @@
+import { getItems, setItems } from './storage.js';
+import createItemElement from './template.js';
+import { postContainer, addButton, addNewInput } from './elements.js';
+
+const refreshList = () => {
+  const items = getItems();
+  postContainer.innerHTML = '';
+
+  items.forEach((item, index) => {
+    const itemElement = createItemElement(item, index);
+    postContainer.appendChild(itemElement);
+  });
+};
+
+const addItem = () => {
+  const description = addNewInput.value;
+
+  const items = getItems();
+  items.push({
+    description,
+    completed: false,
+  });
```

Comment:
- > * This requirement wasn't noted by the previous reviewer because he couldn't run your project however it is a required change for which you are to make corrections 👍 ✨ 
- [ ] Each todo is required to have three properties which are `description, completed & index`, you are currently missing the index. Having the index is a requirement as shown in the image below, because this index is supposed to be reordered everytime a task is deleted, please kindly update your project to include an index 👍 
![re](https://github.com/balatstar/task-list/assets/89055502/fc7dfa73-efdd-46d2-99d8-6fbbfd19d1e4)